ci/docker-pull-or-build-and-push.sh

```diff
@@ -7,6 +7,11 @@ DIRECTORY_HASH=$(git rev-parse HEAD:ci/$image_name)
 export DIRECTORY_HASH
 cd ci/$image_name/
 
+export USER_ID=3001
+export GROUP_ID=3001
```

Comment:
Any specific reason why the change from 3000 to 3001 was needed now? 🤔

Reply:
No reason, just wanted to push new changes to rerun CI tests.